offix/src/main/java/org/aerogear/offix/Offix.kt

```diff
@@ -76,6 +98,39 @@ fun ApolloClient.enqueue(
     this.mutate(mutation).enqueue(apolloCallback)
 }
 
+/*
+ This function is called whenever we get a conflict.
+ Here the mutation with the correct data is retried again.
+ @param mutation: Mutation<Operation.Data, Any, Operation.Variables>
+
+ */
+fun retryConflictedMutation(mutation: Mutation<Operation.Data, Any, Operation.Variables>) {
+
+    /*
+      1. Map the json response String to the Conflict pojo class.
+      2. Extract the correct version from the server data.
+      3. Set the client version to the correct one. (server+1)
+      4. Again make a call with the mutation to the server.
+    */
+    var parsedObject = gson.fromJson(conflcitDetected, ConflictPojo::class.java)
+    var versionUpdated = parsedObject.errors[0].extensions.exception.conflictInfo.serverState.version + 1
```

Comment:
We need to loop through the errors to check for only those errors which are caused due to conflicts.